core/src/main/scala/kafka/tools/ReplicaVerificationTool.scala

```diff
@@ -105,11 +105,18 @@ object ReplicaVerificationTool extends Logging {
                          .describedAs("ms")
                          .ofType(classOf[java.lang.Long])
                          .defaultsTo(30 * 1000L)
+    val helpOpt = parser.accepts("help", "Print usage information.").forHelp()
+    val versionOpt = parser.accepts("version", "Print version information.").forHelp()
```

Comment:
Probably not a big deal, but we could be explicit and say "Print version information and exit."

Reply:
Sure, I'll change it.